src/engine/QueryExecutionContext.cpp

```diff
@@ -34,7 +35,8 @@ QueryExecutionContext::QueryExecutionContext(
       _allocator(std::move(allocator)),
       _sortPerformanceEstimator(sortPerformanceEstimator),
       updateCallback_(std::move(updateCallback)),
-      namedResultCache_(namedResultCache) {}
+      namedResultCache_(namedResultCache),
+      materializedViewsManager_(materializedViewsManager) {}
```

Comment:
maybe add an assertion that all the pointer arguments are not nullptr. (also for the preexisting arguments)